var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -23,6 +23,7 @@ class Gromacs(CMakePackage):
     maintainers = ['junghans', 'marvinbernhardt']
 
     version('master', branch='master')
+    version('2021.rc1', sha256='baab9f9c7a659f0777f0ff06866e88685a4b06d22c0f431f5688a9a559f0a1e1')
```

Comment:
with spack-0.16.0, "2021.rc1" could not be downloaded as the file on ftp.gromacs.org is called gromacs-2021-rc1.tar.gz (2021-rc1 vs. 2021.rc1)

Reply:
Thanks for the feedback. Updated to 2021-rc1.